lib/driver.js

```diff
@@ -137,7 +142,9 @@ class YouiDriver extends BaseDriver {
       browserName: "",
       app: this.caps.app,
       platformVersion: opts.platformVersion,
-      deviceName: opts.deviceName
+      deviceName: opts.deviceName,
+      // Disabling the proxy CommandTimeout
+      newCommandTimeout: 0
     };
```

Comment:
Got it, thanks!

We should probably say instead:

// Disabling the proxy CommandTimeout in the iOS driver since we are now handling it in the Youi Driver